docs/helper.md

```diff
@@ -0,0 +1,94 @@
+# `gpii.handlebars.helper`
+
+This is the base grade for Fluid components that are intended to add new [Handlebars block helpers](http://handlebarsjs.com/block_helpers.html)
+to an instance of [`gpii.express.hb`](handlebars.md).
+
+## Component Invokers
+
+### `{that}.getHelper()`
+
+Each "helper" module is expected to implement a `getHelper` invoker with an invoker that returns a helper function. something like the following:
+
+```
+fluid.registerNamespace("your.helper");
+
+your.helper.getHelperFn = function(that){
+   return function(arg1, arg2) {
```

Comment:
This is rather peculiar as an example - does this represent something that every helper would need to deal with, or is it really just an arbitrary example? Is there any significance to "fn"?